src/browser/api/browser_view.ts

```diff
@@ -108,6 +133,33 @@ export async function setBounds(ofView: OfView, bounds: Rectangle) {
 export function getInfo (ofView: OfView) {
     return getWebContentsInfo(ofView.view.webContents);
 }
+
 export function getCurrentWindow(ofView: OfView) {
     return ofView.target;
+}
+
+export function addEventListener({uuid, name}: Identity, type: string, listener: (...args: any) => void) {
+    const eventString = route.view(type, uuid, name);
+    const errRegex = /^Attempting to call a function in a renderer frame that has been closed or released/;
```

Comment:
better change 'frame' to 'view"  to differentiate it with other entity types.

Reply:
great catch, changed.

Reply:
Well where does this error come from? We need the regex here to match the emitted error